wallet.cc

```diff
@@ -164,13 +164,77 @@ std::string to_string(const PublicWebcash& epk)
     return webcash_string(epk.amount, "public", epk.pk);
 }
 
+// We group outputs based on their use.  There are currently four categories of
+// webcash recognized by the wallet:
+enum HashType : int {
+    // Pre-generated key that hasn't yet been used for any purpose.  To make
```

Comment:
Note that the enum values here are different from the ones used in the python client:
https://github.com/kanzure/webcash/blob/ae2e35c26e9593ed1a41a9191069a7a3cef94452/webcash/walletclient.py#L47

Reply:
Yeah I'll change them to be the same as the reference client. IIRC I was doing something cute with the values such that the enum could be constructed by bit shifting the two fields `mine` (reported as balance in the wallet) and `sweep` (needs to be automatically swept). But that is before you implemented deterministic wallets, and compatibility is more important now.

Reply:
Fixed.